### DISCUSSION TITLE
Linter / formatter for org files

### DISCUSSION THREAD

**Discussion by sarmong** [nvim-orgmode/orgmode#322] [General]
First off, thanks for the great plugin!

I am coming from markdown and still not very familiar with orgmode. 
One thing that I really liked about notetaking with markdown is prettier for formatting and markdown-lint for linting.

Is there such a thing for orgmode?

If no, can I at least somehow autoindent the file?
E.g. I am changing .md file to .org. I change all the headings to use `*` instead of `#`. Now I need to manually indent all the contents to match the heading, which is a bit tedious.

**Comment by kristijanhusak** [comment]
You should be able to get decent indentation/reindentation with `=` mapping. Try doing `gg=G` and see if it works out.

**Reply by sarmong** [reply]
Thanks, that is helpful!
Can I somehow execute it with normal command for all buffer? This doesn't work `:%normal =`

**Reply by kristijanhusak** [reply]
You can do it in two ways:

2 equal signs instead of one:
```
:%normal ==
```

Or go from top to bottom:
```
:normal! gg=G
```

**Reply by sarmong** [reply]
Thanks. I thought using `%` would not move the cursor, but it does. So it still requires executing `<C-o>` to go back.

However, it doesn't indent list items for me, for some reason.

**Reply by sarmong** [reply]
And when using `<C-o>`, the file position in a buffer oftentimes still changes. I run this on `BufWritePre`, so is there a way to indent without changing the position in a buffer?
